.changesets/2508.md

```
@@ -0,0 +1,26 @@
+- fix(dbAuth): Support SimpleWebAuthn v13 (#2508) by @ladderschool
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)